package.json

```diff
@@ -38,4 +85,5 @@
         "nodemon": "^2.0.13",
         "supertest": "^6.1.6"
     }
-}
\ No newline at end of file
+}
+>>>>>>> e7d8d172fd94da2174feae803996e8a1cc17ca4a
```

Comment:
delete conflicts arrows

Reply:
I cant see these line in my footer branch in my computer